llama_hub/singlestore/base.py

```diff
@@ -0,0 +1,68 @@
+"""SingleStore reader."""
+
+from typing import Any, Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from llama_index import download_loader, ListIndex
+
+import pymysql
+
+
+class SingleStoreReader(BaseReader):
+    """SingleStore reader.
+
+    Args:
+        scheme (str): Database Scheme.
+        host (str): Database Host.
+        port (str): Database Port.
+        user (str): Database User.
+        password (str): Database Password.
+        dbname (str): Database Name.
+        table_name (str): Table Name.
+        content_field (str): Content Field.
+        vector_field (str): Vector Field.
+    """
+
+    def __init__(self, scheme: str, host: str, port: str, user: str, password: str, dbname: str, table_name: str, content_field: str = "text", vector_field: str = "embedding"):
+        """Initialize with parameters."""
+        self.scheme = scheme
+        self.host = host
+        self.port = port
+        self.user = user
+        self.password = password
+        self.dbname = dbname
+        self.table_name = table_name
+        self.content_field = content_field
+        self.vector_field = vector_field
+
+        pymysql.install_as_MySQLdb()
+
+        self.DatabaseReader = download_loader('DatabaseReader')
```

Comment:
```
try:
    from llama_hub.utils import import_loader
    
    PandasCSVReader = import_loader("PandasCSVReader")
except ImportError:
    PandasCSVReader = download_loader("PandasCSVReader")
```

Reply:
Is there a reason for using `PandasCSVReader` instead of `DatabaseReader`?

Reply:
oh my bad, that was just an example